edge_computing/image-base-install/ibi-understanding-image-based-install.adoc

```diff
@@ -0,0 +1,92 @@
+* Start the host using the live installation ISO to preinstall the host. 
++
+During this process, the {product-title} installation program installs {op-system-first} to the disk, pulls the image you generated, and precaches release container images to the disk.
+
+* When the installation completes, the host is ready to ship to the remote site for rapid reconfiguration and deployment.
+
+=== Imaged-based deployment for {sno} clusters
+{rh-rhacm-first} and the multicluster engine for Kubernetes operator (MCE) use a hub-and-spoke architecture to manage and deploy {sno} clusters across multiple sites. Using this approach, the hub cluster serves as a central control plane that manages the spoke clusters, which are often remote {sno} clusters deployed at the far edge of the network.
+
+You can define the site-specific configuration resources for an image-based deployment in the hub cluster. The Image Based Install (IBI) Operator uses these configuration resources to reconfigure the preinstalled host at the remote site and deploy the host as a managed {sno} cluster. This approach is especially beneficial for telecommunications providers and other service providers with extensive, distributed infrastructures, where an end-to-end installation at the remote site would be time-consuming and costly.
+
+The following is a high-level overview of the image-based deployment process for hosts preinstalled with an imaged-based installation:
+
+* Define the site-specific configuration resources for the preinstalled host in the hub cluster.
+* Apply these resources in the hub cluster. This initiates the deployment process.
+* The IBI Operator creates a configuration ISO.
+* The IBI Operator boots the target preinstalled host with the configuration ISO attached.
+* The host mounts the configuration ISO and begins the reconfiguration process.
+* When the reconfiguration completes, the {sno} cluster is ready.
+
+As the host is already preinstalled using an image-based installation, a technician can reconfigure and deploy the host in a matter of minutes.
+
+== Image-based installation and deployment key components
+
+The following content describes key components in image-based installation and deployments.
+
+Seed image:: A seed image is an OCI container image generated from a dedicated cluster with the target {product-title} version. 
+
+Seed cluster:: A seed cluster is a dedicated {sno} cluster that you use to create a seed image and is deployed with the target {product-title} version.
+
+{lcao}:: In image-based installations, you can use the {lcao} to generate a seed image, embed the seed image URL in a live installation ISO, and reconfigure the host during a deployment at a remote site.
+
+Image Based Install (IBI) Operator:: The IBI Operator creates a configuration ISO from the site-specific resources you define in the hub cluster, and attaches the configuration ISO to the remote host by using a bare-metal provisioning service.
```

Comment:
Here we should mention the openshift-installer as an alternative to IBIO

Reply:
Added a separate entry to openshift-installer in this context